server/vulnerabilities/nvd/cve.go

```diff
@@ -218,6 +234,14 @@ func checkCVEs(
 							continue
 						}
 
+						if rule, ok := matchingRules.FindMatch(
```

Comment:
Should we rename `matchingRules` to something like `knownNVDBugsRules` or something a bit more descriptive of its purpose?

Reply:
Fixed in https://github.com/fleetdm/fleet/pull/12551/commits/02fa674e13da68ef6a57e706aa4e6fa900ecb2ca